modules/core/src/main/java/org/apache/ignite/internal/processors/cache/persistence/snapshot/IgniteSnapshotManager.java

```diff
@@ -1882,6 +1898,20 @@ static void copy(FileIOFactory factory, File from, File to, long length) {
         return new IgniteFutureImpl<>(cctx.kernalContext().task().execute(taskCls, snpName));
     }
 
+    /**
+     * @param snpName Snapshot name.
+     * @return Snapshot view.
+     */
+    private SnapshotView snapshotViewSupplier(String snpName) {
+        SnapshotMetadata meta = readSnapshotMetadata(snpName,
+            toStringSafe(cctx.kernalContext().discovery().localNode().consistentId()));
```

Comment:
It is possible that the local node also stores snapshots from another node, we should display those as well, not just the local one. 

My suggestion is to change the data supplier to something like this: 
```
() -> localSnapshotNames().stream().map(this::readSnapshotMetadatas).flatMap(List::stream).collect(Collectors.toList()),
```
and replace the current argument of `snapshotViewSupplier` (`String snpName`) to  `SnapshotMetadata meta`